app/views/play/CampaignView.coffee

```diff
@@ -949,10 +949,14 @@ module.exports = class CampaignView extends RootView
   startLevel: (levelElement, courseID=null, courseInstanceID=null) ->
     @setupManager?.destroy()
     levelSlug = levelElement.data 'level-slug'
+    levelOriginal = levelElement.data('level-original')
     session = @preloadedSession if @preloadedSession?.loaded and @preloadedSession.levelSlug is levelSlug
```

Comment:
Let's move session and levelOriginal into the if statement below to make it clearer where they are used.